vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -418,7 +418,7 @@ bool ViewViewManager::UpdateProperty(
       else if (propertyValue.IsNull())
         pViewShadowNode->EnableFocusRing(false);
     } else if (propertyName == "tabIndex") {
-      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Double) {
+      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Int64) {
         auto tabIndex = propertyValue.AsDouble();
```

Comment:
seems wrong to do AsDouble here if the property is an int64.
You can just do AsInt64 and remove the property type check too

Reply:
The representation can also be different when web debugging vs not, so the type check can break things there. +1 to just using AsDouble.

Reply:
Will do. 
@NickGerleman Can you explain how web debugging affects this? Also, you wrote ' +1 to AsDouble' but meant AsInt64, as Alex said, right?